functional_test/pet_store_client.py

```diff
@@ -132,6 +132,18 @@ def signup_user(self, user):
 
         return self.session.request('POST', url, self.headers, json.dumps(user))
 
+    def login_user(self, user):
+        """
+        Logs in an existing user
+        """
+        url = urljoin(self.index_url, '/login')
+
+        response = self.session.post(url,
+                                     headers=self.headers,
+                                     data=json.dumps(user))
+
+        return response
```

Comment:
I attempted to use 
```
        self.headers = dict(self.headers, **response.headers)
```
Here, but just sending that header on each subsequent request does not work. The response headers after login look like:

```
{'Date': 'Tue, 06 Feb 2018 22:33:25 GMT', 'Content-Length': '0', 'Authorization': 'Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOiIxIiwiZXhwIjoxNTE3OTU3MDA1LCJpYXQiOjE1MTc5NTY0MDUsImp0aSI6IjYyNjAxYzljNzczOThmODc1M2ExMDYzMzFkNDBkMjViMWFlYjQ4ZDUyZjNiNjFlMjFkY2E3NzI5YjdiMzY1ODUifQ.vAJEogj6TtCkwU1w4ZGU4Jz812YFZKsM4CaBp6RZErU'}
```